### ISSUE TITLE
bloxshade installer exiting instantly when opening it 

### ISSUE THREAD

**Issue by itsdarky67** [Extravi/Bloxshade#59]
Hello i have a new issue with the bloxshade installer because when i open it it exits instantly 

OS :
windows 11 ghost spectre 24h2 

here is the proof
https://github.com/user-attachments/assets/cee67d5d-8c1d-447c-996c-925ddeeb9189

**Comment by dobrin-glitch** [comment]
same i cant open it too

**Comment by Platon286** [comment]
Me too, I can't open it

**Comment by itsdarky67** [comment]
i found a fix for bloxshade but you can't install community presets ,install an old version of bloxshade but the pro is that it will download the latest shaders

**Comment by Platon286** [comment]
I tried this method but it didn't work

**Comment by Platon286** [comment]
It's still closing

**Comment by Extravi** [comment]
it's because you don't have microsoft edge installed/edge webview2

**Comment by Extravi** [comment]
the bloxshade installer ui uses edge webview2

**Comment by Platon286** [comment]
I have the Microsoft edge, but it's still Close instantly

**Comment by Extravi** [comment]
yes but you dont have edge webview2 installed bc you use a modded version of windows

**Comment by Extravi** [comment]
you can check if you have edge webview2 in control panel under programs

**Comment by Platon286** [comment]
I when install it, I have error 0x8007040c

**Comment by Extravi** [comment]
> I when install it, I have error 0x8007040c

when you use a modded version of Windows that removes important stuff from the operating system it's hard to just add it back

**Comment by Extravi** [comment]
you could probably fix it by deleting the Microsoft edge WebView2 keys that bloxshade checks

**Comment by Platon286** [comment]
How delete keys
